language/llama2-70b/dataset.py

```diff
@@ -0,0 +1,122 @@
+import os
+import time
+import numpy as np
+import torch
+from datasets import load_dataset, load_from_disk
+from transformers import AutoModelForCausalLM, AutoTokenizer
+from torch.nn.functional import pad
+from torch.utils.data import DataLoader
+from typing import Optional, Dict, Sequence
+import io
+#import utils
+import copy
+
+import logging
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger("Llama-70B-Dataset")
+
+import random
+#random.seed(9973)
+
+#TODO Prompt may have to be experimented with. 
+PROMPT_INPUT = "### System:\n{system_prompt}\n### Human:\n{question}\n### Assistant:\n"
```

Comment:
Right. Actually that was just to test functionality. It's not being used now since we have the preprocessing script that generates the  tokenized inputs now. I will remove that.